### PR TITLE
Add Machogu Details to Team section

### PR DESCRIPTION
**Pull request by AbedM15**
Added Machogu's details and links to Team section

### CODE REVIEW

**Comment by muokicaleb** [comment]
Could you change the image dimensions to 600x600

**Comment by AbedM15** [comment]
Done. On my local branch then uploaded to my remote branch. Will I have to
create a new PR for the new changes?

File: index.html [hunk 1]
```diff
@@ -673,16 +673,35 @@ <h4>Steve Fifteen</h4>
...
+                      <a href="https://twitter.com/Machogu_"><i class="bi bi-twitter"></i></a>
+                      <a href=""><i class="bi bi-facebook"></i></a>
+                      <a href=""><i class="bi bi-instagram"></i></a>
+                      <a href="https://www.linkedin.com/in/abednego-machogu-805841215/"><i class="bi bi-linkedin"></i></a>
+                    </div>
+                  </div>
+                  <div class="member-info">
+                    <h4>Machogu</h4>
+                    <span>Software Engineer</span>
+                  </div>
```
Comment (RuthNjeri) [inline index.html]: Is this code supposed to be inside the row div from line 697 below? Ideally where you have removed your name? see suggestion below
```suggestion
```

File: index.html [hunk 2]
```diff
@@ -673,16 +673,35 @@ <h4>Steve Fifteen</h4>
...
-        </div>
+      </div>
           <div class="row"> 
             <!--new change-->
             <!-- awesome code about me-->
             <!--charlesmotaroki@students.uonbi.ac.ke	-->
             <!--wanjaerica@gmail.com	-->
             <!--mboyaivy@gmail.com-->
-            <!--machoguabednego6@gmail.com	-->
+
```
Comment (RuthNjeri) [inline index.html:684]:
```suggestion
              <!--machoguabednego6@gmail.com	-->
              <div class="col-lg-3 col-md-6 d-flex align-items-stretch">
                <div class="member" data-aos="fade-up" data-aos-delay="200">
                  <div class="member-img">
                    <img src="assets/img/team/Machogu.jpeg" class="img-fluid" alt="Machogu's image" width="600" height="600">
                    <div class="social">
                      <a href="https://twitter.com/Machogu_"><i class="bi bi-twitter"></i></a>
                      <a href=""><i class="bi bi-facebook"></i></a>
                      <a href=""><i class="bi bi-instagram"></i></a>
                      <a href="https://www.linkedin.com/in/abednego-machogu-805841215/"><i class="bi bi-linkedin"></i></a>
                    </div>
                  </div>
                  <div class="member-info">
                    <h4>Machogu</h4>
                    <span>Software Engineer</span>
                  </div>
```

**Review by RuthNjeri — CHANGES_REQUESTED** [review]
Good first attempt @AbedM15! There are files on the PR that should not be present like the `.DS_Store`

**Comment by RuthNjeri** [comment]
Let us know if you need any assistance.

**Comment by RuthNjeri** [comment]
Hi @AbedM15, were you able to look at the comments above?

**Comment by AbedM15** [comment]
Hello,

Apologies for the delay but I have finally made the changes as required I
believe.

**Comment by RuthNjeri** [comment]
Hi @AbedM15, I have seen the changes, perhaps you could remove changes not related to this PR? Like these files

Screenshot 2022-03-14 at 10 14 12

Also, you can look into fixing merge conflicts since your PR has one.

**Comment by AbedM15** [comment]
I have removed all .DS_Store files from the folder.

**Comment by RuthNjeri** [comment]
Funny photo! The PR looks good to me!

**Pull request merged**